projects/plugins/jetpack/tests/php/modules/shortcodes/test-class.archives.php

```diff
@@ -1,5 +1,6 @@
 <?php
 
+require_once JETPACK__PLUGIN_DIR . '3rd-party/class.jetpack-amp-support.php';
```

Comment:
This file was included in photon tests, and that change happened in trunk while the PR was open. This is because a couple of unit tests crossover between amp and photon. I marked those as skipped and will be moving those back to jetpack in a separate PR.

Because we have moved all unit test files from jetpack to the package, it started to fail jetpack tests after I merged trunk into this PR.